examples/morphir-elm-projects/evaluator-tests/src/Morphir/Examples/App/DecimalTests.elm

```diff
@@ -12,7 +12,13 @@ decimalFromFloatTest ctx =
     test ctx
         (fromFloat 1.2)
 
-
+{-| Test: Decimal/fromFloat
+expected = 1.2
+-}
+decimalFromFloatTest1 : TestContext -> Decimal
+decimalFromFloatTest1 ctx =
+    test ctx
+        (Decimal.fromFloat 1.2)
```

Comment:
Not sure if we need decimalFromFloatTest1. 

decimalFromFloatTest looks to do the exact same thing with the exact same function -- fromFloat. ```import Morphir.SDK.Decimal as Decimal exposing (..)``` exposes fromFloat (and other Decimal functions) so that we could call it directly.

Reply:
Ok, removed the unnecessary changes.